examples/tensorrt_llm/components/worker.py

```
@@ -14,6 +14,10 @@
 # limitations under the License.
 import asyncio
 import logging
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)